src/shogun/mathematics/graph/ops/ngraph/Add.h

```diff
@@ -28,6 +28,15 @@ namespace shogun {
 		void evaluate() final
 		{
 		}	
+
+		void build_implementation() final
+		{
+			// TODO
+			const auto& node1 = this->m_abstract_node->get_input_nodes()[0];
+			const auto& node2 = this->m_abstract_node->get_input_nodes()[1];
+
+			std::make_shared<ngraph::op::Add>(node1, node2);
```

Comment:
@vigsterkr somehow the nodes need to have access to the ngraph::Node. Which means then that the Node class in Shogun needs to be aware of the operator implementation? Or there is a NGraphNode that can be casted to?